app/shared/footer/FooterContent.js

```diff
@@ -18,7 +19,7 @@ function FooterContent({ t }) {
     case 'ESPOO': {
       return (
         <Grid>
-          <Row>
+          <Row className="flex-container">
```

Comment:
You should never put custom classNames and styles fro Row and Column components.